api/app/src/external/commonwell/patient-shared.ts

```diff
@@ -90,6 +93,40 @@ export async function findOrCreatePerson({
   return { personId, person };
 }
 
+function alertAndReturnMostRecentPerson(
+  commonwellPatientId: string,
+  persons: [CommonwellPerson, ...CommonwellPerson[]],
+  cwReference?: string,
+  context?: string
+): FindOrCreatePersonResponse {
+  const { log } = Util.out(
+    `CW alertAndReturnMostRecentPerson - CW patientId ${commonwellPatientId}`
+  );
+  const personIds = persons.map(getId).flatMap(filterTruthy);
+  const subject = "Found more than one person for patient demographics";
+  const message = idsToAlertMessage(commonwellPatientId, personIds);
+  log(`${subject} - using the most recent one: ${message}`);
+  capture.message(subject, {
+    extra: {
+      action: `Using the most recent one`,
+      commonwellPatientId,
+      personIds,
+      cwReference,
+      context,
+    },
+  });
```

Comment:
since we have a BAA with Sentry - should we just send the person payloads in the capture so we don't have to manually query afterwards?

Reply:
Yep, sounds good 👍